distributions/otelcol-ebpf-profiler/README.md

```diff
@@ -0,0 +1,24 @@
+# OpenTelemetry Collector eBPF Profiling Distribution
+
+This distribution is made specifically to be used as a node agent to gather
+profiles on all processes running on the system.
+
+It contains the [eBPF profiler
+receiver](https://github.com/open-telemetry/opentelemetry-ebpf-profiler) as
+well as a subset of components from [OpenTelemetry Collector
+Core](https://github.com/open-telemetry/opentelemetry-collector) and
+[OpenTelemetry Collector
+Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib).
+
+## Components
+
+The full list of components is available in the [manifest](manifest.yaml).
+
+### Rules for Component Inclusion
```

Comment:
Please add a rule for which processors and extensions are included.

Reply:
I've reworded things, I hope that's what you expected.